src/Microsoft.Framework.OptionsModel/project.json

```diff
@@ -6,6 +6,10 @@
     "Microsoft.Framework.DependencyInjection.Abstractions": "1.0.0-*",
     "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aspnet/options"
```

Comment:
If you add the repository information to you project.json it works out of the box. All it needs in addition to the repo type and the url is the commit ID and the folder, relative to the repo root, where the project it.